vowpalwabbit/parse_args.cc

```diff
@@ -775,7 +775,7 @@ void parse_feature_tweaks(
     {
       if (options.was_supplied("leave_duplicate_interactions"))
       { all.interactions.leave_duplicate_interactions = true; }
-      else
+      else if (!all.logger.quiet)
```

Comment:
should warnings be quiet-eable?

Reply:
all the other ones in parse args are, so this should probably follow along